OrbitCore/SamplingProfiler.cpp

```diff
@@ -359,7 +353,7 @@ void SamplingProfiler::ProcessAddresses() {
   for (const auto& it : m_UniqueCallstacks) {
     CallstackID rawCallstackId = it.first;
     const std::shared_ptr<CallStack> callstack = it.second;
-    CallStack ResolvedCallstack = *callstack;
+    CallStack resolvedCallstack = *callstack;
```

Comment:
resolved_callstack - since you are renaming it anyways

Reply:
Done.